deepchecks/utils/distribution/drift.py

```diff
@@ -89,14 +91,19 @@ def cramers_v(dist1: Union[np.ndarray, pd.Series], dist2: Union[np.ndarray, pd.S
         - 'dist2': Sort by the largest dist2 categories.
         - 'difference': Sort by the largest difference between categories.
         > Note that this parameter has no effect if max_num_categories = None or there are not enough unique categories.
+    from_freqs: bool, default: False
```

Comment:
IMO from_freq is not the right name. For me it implies that the values are already normalized 0-1 or something. Actually if I understood correctly you're saying it already comes as counts rather than raw values, so maybe `from_value_counts` or something along these lines?

Reply:
When doing statistic tests the commonly used term is frequencies https://en.wikipedia.org/wiki/Frequency_(statistics)